src/main/java/com/coveros/selenified/utilities/TestSetup.java

```diff
@@ -306,7 +307,9 @@ public static WebDriver setupDriver(Browser browser,
                 break;
             case OPERA:
                 OperaDriverManager.getInstance().forceCache().setup();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "OperaDriver"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)